SPECS/uclibc-ng/uclibc-ng.spec

```diff
@@ -0,0 +1,195 @@
+%global uclibc_name uClibc-ng
+
+Name:    uclibc-ng
+Version: 1.0.36
+Release: 1%{?dist}
+Summary: C library for embedded Linux
+
+License: LGPLv2
+URL:     http://www.uclibc.org/
+Source0: https://downloads.uclibc-ng.org/releases/%{version}/%{uclibc_name}-%{version}.tar.xz
+Source1: uClibc.config
+
+BuildRequires: gcc
+
+# This package only contains a static library
+%global debug_package %{nil}
+
+%description
+uClibc-ng is a C library for developing embedded Linux systems.
+It is much smaller than the GNU C Library, but nearly all applications
+supported by glibc also work perfectly with uClibc.
+
+%package devel
+Summary: Header files and libraries for uClibc library
+Provides: uclibc-static = %{version}-%{release}
+Provides: uclibc-devel = %{version}-%{release}
+
+%description devel
+uClibc is a C library for developing embedded Linux systems.
+It is much smaller than the GNU C Library, but nearly all applications
+supported by glibc also work perfectly with uClibc.
+This package contains the header files and libraries
+needed for uClibc package.
+
+%prep
+%setup -q -n %{uclibc_name}-%{version}
```

Comment:
Nit: is the `-n` necessary? It seems the name matches with the source. Or am I missing something?

Reply:
Casing is different in case-sensitive scenarios